.github/workflows/all_jobs.yaml

```diff
@@ -12,38 +12,45 @@ env:
   #Windows variables
   python_executable: "C:/hostedtoolcache/windows/Python/3.8.10/x64/python.exe" #Cmake needs this to find python on windows
 
-jobs:
+jobs:  
+
+  container-test-job:
+    name: test
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v4
+
+    - name: Login to Docker Registry
+      uses: docker/login-action@v3
+      with:
+        username: ${{ secrets.PSEE_DOCKER_HUB_USER }}
+        password: ${{ secrets.PSEE_DOCKER_HUB_ACCESS_TOKEN }}
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Build and Push Docker image
+      uses: docker/build-push-action@v5
+      with:
+        context: 
+        file: .github/workflows/Dockerfile.OpenEB 
+        push: true
+        tags: prophesee/openeb:openeb-ubuntu-20.04
```

Comment:
Why not just this ? 
```suggestion
        tags: prophesee/openeb:ubuntu-20.04
```

Reply:
Look at this link : https://hub.docker.com/repository/docker/prophesee/openeb/tags?page=1&ordering=last_updated
The first image is the one that I use and the second one is the suggestion that you put, and you will understand that if we don't specify openeb here we won't know the version of openeb. We know that we are in prophesee/openeb thanks to the repository, but the image will be called only ubuntu-20.04. I want the image to be called openeb-ubuntu-20.04 so we know we are talking about the openeb image